R/utilities.R

```diff
@@ -2310,7 +2310,11 @@ collate_sv_results = function(sample_table,
 #' # install_github("morinlab/ggsci")
 #'
 get_gambl_colours = function(classification = "all",
-                             alpha = 1,as_list=FALSE){
+                             alpha = 1,
```

Comment:
Do you want to add descriptions of these new parameters so they can be properly documented? If not, I can do it in my oveRhaul.